autoware_map_msgs/srv/GetDifferentialLanelet2Map.srv

```diff
@@ -0,0 +1,9 @@
+# Osm file ids which are selected to be loaded
+int32[] vector_map_file_ids
+
+---
+# Header
+std_msgs/Header header
+
+# Newly loaded Lanelet map
+autoware_auto_mapping_msgs/HADMapBin differential_map
```

Comment:
Let's use LaneletMapBin message instead of HADMapBin: https://github.com/autowarefoundation/autoware_msgs/blob/main/autoware_map_msgs/msg/LaneletMapBin.msg

Reply:
I would like to apologize and ask again about this matter. I can't recall what needs to be done to avoid issues with planning and mapping components when changing the old message type to the new message type. Instead of updating all planning and mapping components, should I only change the message type in autoware_map_provider from the old to the new one? As a result, the service call will return the new message type, but autoware_map_provider will continue to publish the old message type?

Reply:
I would like you to modify the message type in the service for now. The rest can stay as is.
![image](https://github.com/autowarefoundation/autoware_msgs/assets/43976834/ed356cc8-14eb-420b-9976-aa5781fef581)

Reply:
I remove the autoware_mapping_msg dependency, and I add  autoware_map_msg/LaneletMapBin to service 👍🏽 👍🏽